tests/failpoint_tests.rs

```diff
@@ -27,14 +30,17 @@ async fn txn_optimistic_heartbeat() -> Result<()> {
     let mut heartbeat_txn = client
         .begin_with_options(
             TransactionOptions::new_optimistic()
-                .heartbeat_option(HeartbeatOption::FixedTime(Duration::from_secs(1))),
+                .heartbeat_option(HeartbeatOption::FixedTime(Duration::from_secs(1)))
+                .drop_check(CheckLevel::Warn),
```

Comment:
Because it's not stable in CI ? Suggest to send an issue for this, and leave comments here about that the warning level for drop check should be removed later.